biosynseq/generate_test.py

```diff
@@ -0,0 +1,11 @@
+"""Test module for generate.py."""
+
+from biosynseq.generate import fasta_to_embeddings, generate_fasta
+import numpy as np
+
+def test_generate_fasta() -> None:
+
+    return None
+
+def test_fasta_to_embeddings() -> np.ndarray:
```

Comment:
In general, tests that require GPUs can be difficult. Don't worry too much about testing this function at the moment, we can discuss more later. For now just make sure it runs correctly on lambda.